tests/dags/providers/provider_api_scripts/test_cleveland_museum_of_art.py

```diff
@@ -49,47 +55,82 @@ def test_build_query_param_with_givens():
 
 
 def test_get_image_type_web():
+    response_json = _get_resource_json("response_success.json")
     image_data = _get_resource_json("image_type_web.json")
-    actual_url, actual_key = clm._get_image_type(image_data)
+    data = response_json["data"][0]
+    data["images"] = image_data
+    actual_image = clm._handle_batch_item(data)
+
     expected_url = "https://openaccess-cdn.clevelandart.org/1335.1917/1335.1917_web.jpg"
-    expected_key = "web"
+    expected_width = 1263
+    expected_height = 775
+    expected_filetype = "jpg"
+    expected_filesize = 716717
 
-    assert actual_url == expected_url
-    assert actual_key == expected_key
+    assert actual_image["image_url"] == expected_url
+    assert actual_image["width"] == expected_width
+    assert actual_image["height"] == expected_height
+    assert actual_image["filetype"] == expected_filetype
+    assert actual_image["filesize"] == expected_filesize
```

Comment:
Nit: I think it's totally fine to make assertions in the form of `assert actual_image["width"] == 1263` rather than setting them to a separate variable, especially if we aren't reusing that variable anywhere else 🙂 I recognize that's a common pattern throughout the tests so maybe not something that needs to be changed here!